eng/doc/fips/README.md

```diff
@@ -360,9 +360,11 @@ This feature does not require any additional configuration, but it only works wi
 
 ### TLS with FIPS-approved settings
 
-The Go TLS stack will automatically use OpenSSL crypto primitives when running in FIPS mode. Yet, the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
+The Go TLS stack will automatically use OpenSSL crypto primitives. Yet, the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
 
-A program can import the `crypto/tls/fipsonly` package to configure the Go TLS stack so it is compliant with these restrictions. The configuration is done by an `init()` function, so only importing it is necessary:
+Since Go 1.22, the Microsoft Go runtime automatically enforces that `crypto/tls` and `crypto/x509` only use FIPS-approved settings when running in FIPS mode. This differs from upstream's BoringCrypto backend, which requires you to import `crypto/tls/fipsonly` to apply the FIPS-mandated restrictions. The Microsoft Go crypto backends do this automatically to reduce the source code changes necessary to produce a FIPS-compliant Go application.
```

Comment:
> The Microsoft Go crypto backends do this automatically to reduce the source code changes necessary to produce a FIPS-compliant Go application.

Hmm. Probably worth mentioning that this makes a single Go binary able to run with/without FIPS-restricted TLS, too.

But I wonder if this concept in general is worth its own new Features section, e.g. first we have "No code changes required" (which is arguably truer once we have this feature 😄), then "Build one binary for FIPS and non-FIPS" or something like htat. I take this for granted now, but for teams that are poking through our docs to figure out if it works for their scenario, I think this is worth highlighting in itself.

Reply:
> Hmm. Probably worth mentioning that this makes a single Go binary able to run with/without FIPS-restricted TLS, too.

Done!

> But I wonder if this concept in general is worth its own new Features section...

IMO this is why we have `Usage: Common configurations`, and in there we already explain that FIPS or no-FIPS is a runtime decision unless `-tags=requirefips` is set.

Reply:
Yeah, reading that section again, I agree it's enough.